docs/development/releasing.rst

```diff
@@ -442,6 +460,46 @@ be fixed soon to a new release milestone.  If the upcoming bug fix release is
 issues that you don't expect to be fixed in time for 'v0.2.2'.
 
 
+.. _helpers-release-info:
+
+Coordinating Astropy and astropy-helpers Releases
+-------------------------------------------------
+
+A bit more initial effort is required for an Astropy release that has a
+corresponding astropy-helpers release.  The main reason for this more complex
+procedure is to allow the Astropy core to be tested aginst the new helpers
+before anything is released.  Hence the following procedure should be added
+to the beginning of the above procedure when this is required.
+
+#. In `astropy-helpers`, create a new release branch "release-<version>".
```

Comment:
I understand now why this was done - if the subsequent PR into master fails for any reason, we can easily re-edit the history of the release branch, but we can't do that for the maintenance or master branches. So I withdraw my objection to this procedure.